zebra-rpc/src/methods.rs

```diff
@@ -526,6 +549,55 @@ where
         }
         .boxed()
     }
+
+    fn get_address_tx_ids(
+        &self,
+        addresses: Vec<String>,
+        start: u32,
+        end: u32,
+    ) -> BoxFuture<Result<Vec<String>>> {
+        let mut state = self.state.clone();
+        let mut response_transactions = vec![];
+        let start = Height(start);
+        let end = Height(end);
+
+        async move {
+            if start > end {
+                return Err(Error::invalid_params(
+                    "End height should be at least start height",
+                ));
+            }
+
+            let mut valid_addresses = vec![];
+            for address in addresses {
+                let valid_address: Address = address.parse().map_err(|_| {
+                    Error::invalid_params(format!("Provided address is not valid: {}", address))
+                })?;
+                valid_addresses.push(valid_address);
+            }
```

Comment:
Can probably be done in one line, but i was not able to make it work yet ...

Reply:
https://github.com/ZcashFoundation/zebra/pull/4062/commits/b0cce84289f83e71c9d6b45990cac4519ad89510